projects/driver_controls/inc/magnetic_sensor.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+// Driver for the TLV493D magnetic sensor
+// I2C must be enabled
```

Comment:
It's nice to include all the dependencies so we don't need to trawl through all the module dependencies.